Essentials/src/main/java/com/earth2me/essentials/commands/Commandworth.java

```diff
@@ -80,15 +79,19 @@ private BigDecimal itemWorth(final CommandSource sender, final User user, final
 
         final BigDecimal worth = ess.getWorth().getPrice(ess, is);
         if (worth == null) {
-            throw new Exception(tl("itemCannotBeSold"));
+            throw new TranslatableException("itemCannotBeSold");
         }
 
         if (amount < 0) {
             amount = 0;
         }
 
         final BigDecimal result = worth.multiply(BigDecimal.valueOf(amount));
-        sender.sendMessage(MaterialUtil.getDamage(is) != 0 ? tl("worthMeta", is.getType().toString().toLowerCase(Locale.ENGLISH).replace("_", ""), MaterialUtil.getDamage(is), NumberUtil.displayCurrency(result, ess), amount, NumberUtil.displayCurrency(worth, ess)) : tl("worth", is.getType().toString().toLowerCase(Locale.ENGLISH).replace("_", ""), NumberUtil.displayCurrency(result, ess), amount, NumberUtil.displayCurrency(worth, ess)));
+        if (MaterialUtil.getDamage(is) != 0) {
+            sender.sendTl("worthMeta", is.getType().toString().toLowerCase(Locale.ENGLISH).replace("_", ""), MaterialUtil.getDamage(is), NumberUtil.displayCurrency(result, ess), amount, NumberUtil.displayCurrency(worth, ess));
+        } else {
+            sender.sendTl("worth", is.getType().toString().toLowerCase(Locale.ENGLISH).replace("_", ""), NumberUtil.displayCurrency(result, ess), amount, NumberUtil.displayCurrency(worth, ess));
+        }
```

Comment:
```suggestion
        final String itemName = is.getType().toString().toLowerCase(Locale.ENGLISH).replace("_", "");
        if (MaterialUtil.getDamage(is) != 0) {
            sender.sendTl("worthMeta", itemName, MaterialUtil.getDamage(is), NumberUtil.displayCurrency(result, ess), amount, NumberUtil.displayCurrency(worth, ess));
        } else {
            sender.sendTl("worth", itemName, NumberUtil.displayCurrency(result, ess), amount, NumberUtil.displayCurrency(worth, ess));
        }
```
I hope this compiles